code/game/machinery/iv_drip.dm

```diff
@@ -56,6 +59,10 @@
 		if(internal_list_reagents)
 			reagents.add_reagent_list(internal_list_reagents)
 	interaction_flags_machine |= INTERACT_MACHINE_OFFLINE
+	AddElement( \
+		/datum/element/contextual_screentip_bare_hands, \
+		rmb_text = "Detach / Eject / Configure", \
```

Comment:
Surely it doesn't do all at the same time, every time?

Reply:
If what RMB does is contextual (it definitely is, can't eject something you don't have), then you can't use this element.

You instead want to override `add_context`, and call `register_context` in Initialize. Ctrl+shift+F for `/add_context` to see examples.

Reply:
You want the RMB text to change depending on what it's going to do.